Packs/CommonPlaybooks/Playbooks/playbook-Threat_Hunting_-_Set_Indicators_README.md

```diff
@@ -0,0 +1,106 @@
+This playbook normalizes a variety of indicators to facilitate their use within threat hunting and detection playbooks, as well as reduce the possibility of task errors.
+Using this playbook as a sub-playbook in threat hunting and detection playbooks is particularly useful if inputs are manually entered.
```

Comment:
I think we can remove this sentence and add the first part to the sentence after (Using this playbook as a sub-playbook in threat hunting and detection playbooks to remove duplications...... )